### PR TITLE
[dev] Changed kafka2ftp license to Apache 2.0, remove old library and add example

### PR DESCRIPTION
**Pull request by mprevide**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR? (Link to an issue, e.g. #99999)** 

* **Other information**:
#1879

### CODE REVIEW

**Review by thiagodpf — APPROVED** [review]

File: connector/kafka2ftp/app/KafkaFTPConsumers.js [hunk 1]
```diff
@@ -13,21 +11,21 @@ class KafkaFTPConsumers {
    *
    */
   constructor() {
-    logger.debug('constructor: Instance Kafka', TAG);
-    this.consumer = new ConsumerBackPressure(consumerConfig);
+    logger.debug('constructor: Instance Kafka');
+    this.consumer = new Consumer({ 'kafka.consumer': consumerConfig });
```
Comment (rascaraficci) [inline connector/kafka2ftp/app/KafkaFTPConsumers.js:15]: Shouldn't we pass the topic settings too (second parameter of the constructor)?

**Review by rascaraficci — COMMENTED** [review]
I left just one comment.

File: connector/kafka2ftp/app/KafkaFTPConsumers.js (hunk 1 above)
Comment (mprevide) [reply connector/kafka2ftp/app/KafkaFTPConsumers.js:15]: The configurations that existed were only of the consumer type and the ConfigManager was not used, so currently no topic configuration is passed. Do you think I should put topic settings?

Comment (rascaraficci) [reply connector/kafka2ftp/app/KafkaFTPConsumers.js:15]: Let's keep this way.

**Review by rascaraficci — APPROVED** [review]
It's OK to me!

**Pull request merged**